contrib/babelfishpg_tsql/src/fts.c

```diff
@@ -13,6 +14,13 @@ babelfish_fts_rewrite(PG_FUNCTION_ARGS)
     char* input_str = text_to_cstring(input_text);
     char* translated_query;
     text* result_text = NULL; // Initialize result_text to NULL
+
+    if (!pltsql_allow_fulltext_parser)
+    {
+        ereport(ERROR,
+                (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+                errmsg("Full Text Search is not yet supported.")));
```

Comment:
Are we sure about this error msg ? It's not supported ?

Reply:
This is a fail-safe, as we are waiting for appsec review. The idea is that until the parser is approved by appsec, it will be controlled by a superuser-only GUC. Customers who wish to experiment with full text search in red panda can submit a request for CP to override the GUC to allow them to use full text search features. We will remove this GUC in the next release after Red Panda assuming that it works well.